modules/special-rules/pages/flak.adoc

```diff
@@ -1,2 +1,62 @@
 = Flak
 
+These units are specialised to shoot down enemy flyers and drop pods.
+Some use very sophisticated targeting devices and fast, accurate weapons systems.
+Others simply fill the air with so many munitions that they can hardly fail to get some hits.
+
+---
+
+{flak} units can attack flyers in one of two different ways, depending on what orders you give to their detachment.
+
+[IMPORTANT]
+.Flak and drop pods
+====
+{flak} units may fire on drop pods as if they were flyers, against the Armour value of the transported unit.
```

Comment:
```suggestion
{flak} units may shoot at drop pods as if they were flyers.
For your rolls to hit, use the Armour value of the transported unit.
```
Just making it a bit more explicit - the use of 'against' is slightly ambiguous.
Also note [fire; shoot](https://epicremastered.com/contributors-guide/a-to-z-of-style.html#_fire_shoot)